cinn/auto_schedule/search_space/auto_gen_rule/multi_level_tiling_test.cc

```diff
@@ -166,5 +166,91 @@ TEST(MulitLevelTile, MatrixMultiply) {
   VLOG(6) << expr_str;
 }
 
+TEST(MultiLevelTile, SimpleLoops2) {
```

Comment:
```suggestion
TEST(MultiLevelTile, ApplyOnBlock) {
```
新case名称统一为ApplyOnBlock，提高可读性，或者在原有case下添加新的assert判断，并注释说明测试哪部分功能